staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go

```diff
@@ -124,8 +137,8 @@ func UpdateResource(r rest.Updater, scope RequestScope, admit admission.Interfac
 				ctx,
 				name,
 				rest.DefaultUpdatedObjectInfo(obj, transformers...),
-				withAuthorization(rest.AdmissionToValidateObjectFunc(admit, staticAdmissionAttributes), scope.Authorizer, createAuthorizerAttributes),
-				rest.AdmissionToValidateObjectUpdateFunc(admit, staticAdmissionAttributes),
+				rest.AdmissionToValidateObjectFunc(admit, admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Create, userInfo)),
```

Comment:
why is `withAuthorization` dropped?

Reply:
I think it was added recently. Maybe just a rebase error....

Reply:
@sttts Yep. Nice catch, or it must be a bad mistake I believe. 😅